start_esp/start_esp.py

```diff
@@ -577,11 +577,13 @@ def make_argparser():
     {template}.''')
 
     parser.add_argument('--client_real_ip_position', default=None, help='''
-    Defines the position of the client IP address. If the value is not defined
-    or set to 0, then client_real_ip_header will be ignored.
-    Positive number(>=1) will take the IP address from left to right and
-    negative number (<= -1)will be take it from the right to left''')
-
+    Defines the position of the client IP address.
+    If the value is >= 0, it will take the IP address from left to right
```

Comment:
also say:  only used when flag --client_ip_header is specified

Reply:
Applied